### PR TITLE
Mouthwash

### PR DESCRIPTION
**Pull request by DreamySkrell**
ready for review

### CODE REVIEW

**Comment by DreamySkrell** [comment]
!wip

**Comment by DreamySkrell** [comment]
!review

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 1]
```diff
@@ -958,18 +958,26 @@
...
 	can_hold = list(/obj/item/reagent_containers/toothpaste,
-					/obj/item/reagent_containers/toothbrush)
+					/obj/item/reagent_containers/toothbrush,
+					/obj/item/reagent_containers/food/drinks/flask/vacuumflask/mouthwash,
+					)
 
 	starts_with = list(/obj/item/reagent_containers/toothpaste = 1,
-					/obj/item/reagent_containers/toothbrush = 1)
+					/obj/item/reagent_containers/toothbrush = 1,
+					/obj/item/reagent_containers/food/drinks/flask/vacuumflask/mouthwash = 1,
```
Comment (Alberyk) [inline code/game/objects/items/weapons/storage/boxes.dm:967]:
```suggestion
					/obj/item/reagent_containers/food/drinks/flask/vacuumflask/mouthwash = 1
```

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 2]
```diff
@@ -958,18 +958,26 @@
...
+					)
 
 /obj/item/storage/box/toothpaste/green
 	starts_with = list(/obj/item/reagent_containers/toothpaste = 1,
-					/obj/item/reagent_containers/toothbrush/green = 1)
+					/obj/item/reagent_containers/toothbrush/green = 1,
+					/obj/item/reagent_containers/food/drinks/flask/vacuumflask/mouthwash = 1,
```
Comment (Alberyk) [inline code/game/objects/items/weapons/storage/boxes.dm:973]:
```suggestion
					/obj/item/reagent_containers/food/drinks/flask/vacuumflask/mouthwash = 1
```

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 3]
```diff
@@ -958,18 +958,26 @@
...
+					)
 
 /obj/item/storage/box/toothpaste/red
 	starts_with = list(/obj/item/reagent_containers/toothpaste = 1,
-				/obj/item/reagent_containers/toothbrush/red = 1)
+				/obj/item/reagent_containers/toothbrush/red = 1,
+					/obj/item/reagent_containers/food/drinks/flask/vacuumflask/mouthwash = 1,
```
Comment (Alberyk) [inline code/game/objects/items/weapons/storage/boxes.dm:979]:
```suggestion
					/obj/item/reagent_containers/food/drinks/flask/vacuumflask/mouthwash = 1
```

**Review by Alberyk — DISMISSED** [review]

File: code/game/objects/items/weapons/storage/boxes.dm (hunk 1 above)
Comment (DreamySkrell) [reply code/game/objects/items/weapons/storage/boxes.dm:967]: same as previous

File: code/game/objects/items/weapons/storage/boxes.dm (hunk 2 above)
Comment (DreamySkrell) [reply code/game/objects/items/weapons/storage/boxes.dm:973]: same as previous

File: code/game/objects/items/weapons/storage/boxes.dm (hunk 3 above)
Comment (DreamySkrell) [reply code/game/objects/items/weapons/storage/boxes.dm:979]: same as previous

**Review by NonQueueingMatt — APPROVED** [review]

**Review by FluffyGhoster — APPROVED** [review]

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 4]
```diff
@@ -958,18 +958,26 @@
 
 /obj/item/storage/box/toothpaste
 	can_hold = list(/obj/item/reagent_containers/toothpaste,
-					/obj/item/reagent_containers/toothbrush)
+					/obj/item/reagent_containers/toothbrush,
+					/obj/item/reagent_containers/food/drinks/flask/vacuumflask/mouthwash,
```
[the remark this answers is not in the document]
Comment (FluffyGhoster) [reply code/game/objects/items/weapons/storage/boxes.dm:962]: Dreamy is right

**Pull request merged**